core/metrics-core-service/src/main/java/org/hawkular/metrics/core/service/DataAccess.java

```diff
@@ -67,21 +67,22 @@
 
     Observable<Row> findAllMetricsInData();
 
-    Observable<Integer> insertGaugeDatas(Observable<Metric<Double>> gauges, Function<MetricId, Integer> ttlFunc);
+    Observable<Integer> insertGaugeDatas(Observable<Metric<Double>> gauges,
+            Function<MetricId<Double>, Integer> ttlFunc);
 
     Observable<Integer> insertGaugeData(Metric<Double> metric);
 
     Observable<Integer> insertGaugeData(Metric<Double> metric, int ttl);
 
-    Observable<Integer> insertStringDatas(Observable<Metric<String>> strings, Function<MetricId, Integer>
-            ttlFetcher, int maxSize);
+    Observable<Integer> insertStringDatas(Observable<Metric<String>> strings,
```

Comment:
That is configured in [hawkular/hawkular-build-tools](https://github.com/hawkular/hawkular-build-tools) project. I just use the configuration from there.